src/main/resources/resource/Intro/InMoov01_start.py

```diff
@@ -1,8 +0,0 @@
-#########################################
-# InMoov01_start.py
```

Comment:
Deletion of this is fine in my eyes since all it does is start the service, but probably should let users know somehow in case they execute it manually

Reply:
how ? :shrug: